Project/Source/Components/UI/ComponentImage.cpp

```diff
@@ -26,6 +26,7 @@
 #define JSON_TAG_TEXTURE_TEXTUREID "TextureId"
 #define JSON_TAG_COLOR "Color"
 #define JSON_TAG_ALPHATRANSPARENCY "AlphaTransparency"
+#define JSON_TAG_ENABLED "Enabled"
```

Comment:
Change to the same name as the attribute so it doesn't confuse with "active"

Reply:
It's been removed.